packages/itwinui-css/src/tile/tile.scss

```diff
@@ -121,6 +138,10 @@
     text-overflow: ellipsis;
   }
 
+  .iui-tile-uploading {
```

Comment:
Naming is weird.. why not just `iui-informational`

Reply:
Fixed this